app/javascript/src/components/OrganizationSetup/index.tsx

```diff
@@ -105,10 +107,18 @@ const OrganizationSetup = () => {
   ) => {
     setStepNoOfLastSubmittedForm(currentStep);
     const payload = generatePayload(companyDetails, financialDetails);
-    const res = await companiesApi.create(payload);
 
-    if (res?.status == 200) {
-      navigate(Paths.SIGNUP_SUCCESS);
+    try {
+      const res = await companiesApi.create(payload);
+      if (res?.status == 200) {
+        navigate(Paths.SIGNUP_SUCCESS);
+      }
+    } catch (error) {
+      Toastr.error(
+        error.response?.data?.errors ||
+          error.response?.data?.error ||
+          error.response?.data?.notice
+      );
```

Comment:
```suggestion
      const { errors, error, notice } = error?.response?.data || {};
      const errorMessage = errors || error || notice;
      
      Toastr.error(errorMessage);
```

I have not test this but can you check if this works and move error object to errorMessage variable?

Reply:
Done